pxtlib/main.ts

```diff
@@ -395,6 +395,7 @@ namespace pxt {
         teachertoolUrl?: string; // "/beta---eval"
         isStatic?: boolean;
         verprefix?: string; // "v1"
+        ocvEnabled: boolean;
```

Comment:
I think if you define this as an optional field (ocvEnabled?) then you won't need to go in and add it to all those other files  (like test.spec.ts)

Reply:
+1